scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/attestation/GramineQuoteGeneratorImpl.scala

```diff
@@ -17,20 +17,20 @@
 package com.intel.analytics.bigdl.ppml.attestation
 
 import org.apache.logging.log4j.LogManager
-import java.io.BufferedOutputStream;
+import java.io.{BufferedOutputStream, BufferedInputStream};
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.FileOutputStream;
+import scala.collection.Iterator
 
 /**
  * QuoteGenerator for Gramine (https://github.com/gramineproject/gramine)
  */
 class GramineQuoteGeneratorImpl extends QuoteGenerator {
 
   val logger = LogManager.getLogger(getClass)
-
-  val userReportPath = "/dev/attestation/user_report_data"
-  val quotePath = "/dev/attestation/quote"
+  val UserReportPath = "/dev/attestation/user_report_data"
```

Comment:
Change to all uppercase .e.g., MAGIC_NUM

Reply:
Thanks for the feedback. The style of the two constants are changed.